migrations/2023-07-17-111427_add-fraud-check-table.sql/up.sql

```diff
@@ -0,0 +1,34 @@
+-- Your SQL goes here-- Your SQL goes here
+CREATE TYPE "FraudCheckType" AS ENUM (
+    'pre_frm',
+    'post_frm'
+);
+
+CREATE TYPE "FraudCheckStatus" AS ENUM (
+    'fraud',
+    'manual_review',
+    'pending',
+    'legit',
+    'transaction_failure'
+);
+
+CREATE TABLE fraud_check (
+    id SERIAL PRIMARY KEY,
+    frm_id VARCHAR(255) NOT NULL UNIQUE,
```

Comment:
for internally created ids use VARCHAR(64) and externally receiving ids use VARCHAR(124)